projects/showcase/cypress/e2e/language-switch.cy.ts

```diff
@@ -1,49 +1,55 @@
-import { E2eDriver } from '../support/e2e-driver';
+import { E2eDriver } from "../support/e2e-driver";
 
-describe('Language switch', () => {
+describe("Language switch", () => {
   context(
-    'Given I am on the Language Switch showcase and French is selected',
+    "Given I am on the Language Switch showcase and French is selected",
     () => {
-      let driver = new E2eDriver();
+      let { beforeAndAfter, when, get, then } = new E2eDriver();
+      let initialImageSnapshot: any;
+
+      beforeAndAfter();
 
       beforeEach(() => {
-        driver
-          .visitMapPage('/demo/language-switch')
-          .waitForMapToIdle()
-
-          // Start in French for the baseline
-          .when.clickLanguageButton('French')
-          .waitForMapToIdle()
-          .takeImageSnapshot();
+        ({ when, get, then } = new E2eDriver());
```

Comment:
I think there's a redundant `)` and `(` here, right?

Reply:
Also this is done in the context as well, so I'm not sure it's needed here...?

Reply:
This re-assigning of destructed attributes, will not compile without ()
I don't see where it's done in the context, can you give me a permalink to line?

Reply:
A few lines above this line.